### PR TITLE
perf-tuning: add wrong-index-solution

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Add wrong-index-solution.

**Note:** CI fails because a link anchor (Count-Min Sketch) waits on the translation of https://github.com/pingcap/docs-cn/pull/3223.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3302, https://github.com/pingcap/docs-cn/pull/3934
- Other reference link(s):

### CODE REVIEW

File: wrong-index-solution.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+The low health state means TiDB has not performed the`ANALYZE` statement for a long time. You can update the statistics by running the `ANALYZE` command. After the update, if the wrong usage of indexes still persists, refer to the next section.
+
+## Near 100% health state
+
+The near 100% health state suggests that the `ANALYZE` statement is just completed or was completed a short time ago. In this case, the wrong index issue might be related to TiDB's estimation logic for the number of rows.
+
+For equivalence queries, the cause might be [Count-Min Sketch](/statistics.md#count-min-sketch). You can check whether Count-Min Sketch is the cause and take corresponding solutions. 
+
+If the cause above does not apply to your problem, you can force-select indexes by executing `USE_INDEX` or `use index` (see [USE_INDEX](/optimizer-hints.md#use_indext1_name-idx1_name--idx2_name-) for details). In addition, you can also non-intrusively change the query behavior by using [SQL Plan Management](/sql-plan-management.md).
```
Comment (yikeke) [inline wrong-index-solution.md:23]:
```suggestion
If the cause above does not apply to your problem, you can force-select indexes by using the `USE_INDEX` or `use index` optimzer hint (see [USE_INDEX](/optimizer-hints.md#use_indext1_name-idx1_name--idx2_name-) for details). Also, you can change the query behavior by using [SQL Plan Management](/sql-plan-management.md) in a non-intrusive way.
```

**Comment by yikeke** [comment]
Please fix the lint. @ran-huang

**Comment by ran-huang** [comment]
> Please fix the lint. @ran-huang

@yikeke Thanks, but the lint has to wait until this PR (https://github.com/pingcap/docs-cn/pull/3223) is translated by Joyce.

**Comment by yikeke** [comment]
> > Please fix the lint. @ran-huang
> 
> @yikeke Thanks, but the lint has to wait until this PR ([pingcap/docs-cn#3223](https://github.com/pingcap/docs-cn/pull/3223)) is translated by Joyce.

Waiting on https://github.com/pingcap/docs/pull/3332. Could you please review #3332? @ran-huang

**Comment by ran-huang** [comment]
@winoros PTAL. Thanks~

**Pull request merged**